packages/medical-domain/src/domains/repositories/medical/MedicalDataService.ts

```diff
@@ -126,6 +126,7 @@ class MedicalDataService {
         } else {
           message = String(error)
         }
+        console.log(raw)
```

Comment:
To be removed

Reply:
logging only error doesn't give many context... I'll change the log to include both error and data (with only one console.log).